package.json

```diff
@@ -230,7 +230,7 @@
 		"autoprefixer": "postcss -u autoprefixer -r --no-map --config packages/calypso-build/postcss.config.js",
 		"postcss": "postcss -r --config packages/calypso-build/postcss.config.js",
 		"prebuild": "npm run -s install-if-deps-outdated",
-		"build": "npm run build-static && npm run build-packages && npm run build-css && run-p -s build-devdocs:* && run-p -s build-server build-client-if-prod build-client-if-desktop",
+		"build": "npm run build-static && npm run build-packages && npm run build-css && run-p -s build-devdocs:* && run-p -s build-server build-client-if-prod build-client-if-desktop && npm run components:storybook:build",
```

Comment:
The `components:storybook:build` script can be built in parallel (`run-p ...`) with other scripts. Either in the `build-devdocs:*` batch or in the `build-server/client` one.

That might speed up the build a little.